RunOneSim.py

```diff
@@ -22,14 +22,43 @@
 
 #====================================== HELPER =================================================
 
-def runSim(simSetting, simUI=None):
+def runOneSim(simSetting, simUI=None):
     '''
     Run a single simulation. Finishes when map is complete (or mapping times out).
     '''
 
     # ======================== setup
 
-    log.info("Simulation started")
+    # log
+    log.info(
+        "run {}/{} starting ...".format(
+            simSetting['seed'],
+            simSetting['numberOfRuns'],
+        )
+    )
+
+    # setup data collection
+    dataCollector = DataCollector.DataCollector()
+    log_dir = "./logs"
+    os.makedirs(log_dir, exist_ok=True)
+    dataCollector.setFileName(
```

Comment:
I recommend you specify just the filename e.g. `poipoi` here. In `dataCollector.setFileName()`, add the path and the suffix.

Reply:
can we open a separate issue for this?